ocaml/xapi-idl/lib/dune

```diff
@@ -12,6 +12,7 @@
    message-switch-unix 
    mtime
    mtime.clock.os
+   open-uri
```

Comment:
for consistency should it be `xapi-open-uri`?

Reply:
Hmm yes, that's probably better.

Reply:
Done.